src/main/java/com/salesforce/dataloader/security/EncryptionUtil.java

```diff
@@ -75,12 +76,13 @@ public static String bytesToText(byte[] bytes) {
 
 
     private static void printUsage() {
-        LOGGER.info("\nUtility to encrypt a string based on a static or a provided key");
-        LOGGER.info("Options (mutually exclusive - use one at a time): \n"
-                + "\t-e <plain text> <Path to keyfile>                Encrypt a plain text value, keyfile path\n)"
-                + "\t-d <encryptText> <Path to keyfile>               Decrypt an encrypted text back to plain text value using keyfile\n)"
-                + "\t-k [Path to Keyfile]                             Generate keyfile with optional keyfile path\n"
-        );
+        String usage = "\nUtility to encrypt a string based on a static or a provided key\n"
+                        + "Options (mutually exclusive - use one at a time): \n"
+                        + "\t-e <plain text> <Path to keyfile>                Encrypt a plain text value, keyfile path\n"
+                        + "\t-d <encryptText> <Path to keyfile>               Decrypt an encrypted text back to plain text value using keyfile\n"
+                        + "\t-k [Path to Keyfile]                             Generate keyfile with optional keyfile path\n";
+
+        System.out.println(usage);
```

Comment:
DL is printing out the log lines in console because our default log4j appender is set to to stdout but if user changes the log level to something less from INFO like (WARN, ERROR or FATAL), that usage will not show in the console. Whereas, System.out will write out to console regardless of log level